src/tests/fixtures/users.fixture.js

```diff
@@ -59,6 +59,24 @@ export const loggedInUser = {
   updatedAt: new Date(),
 };
 
+export const unverifiedUser = {
+  id: 80,
```

Comment:
Is there a reason why this id is not auto-generated?

Reply:
@adesege Ok solved on my faker data but left it intact on other team mates faker data. They said that they have implemented it in theirs.